.github/workflows/nightly.yml

```diff
@@ -46,6 +46,6 @@ jobs:
     - name: Deploy
       if: success()
       env:
-        PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        PYPI_USERNAME: allennlp
```

Comment:
Using a secret for this causes any mention of "allennlp" to be starred out with "***" in the build logs which is annoying